.github/workflows/lint-i18n.yml

```diff
@@ -0,0 +1,135 @@
+name: Lint I18N
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 12 * * 1-5'
+  push:
+    paths:
+      - '.github/workflows/lint-i18n.yml'
+    branches:
+      - main
+  pull_request:
+    paths:
+      - '.github/workflows/lint-i18n.yml'
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Cancel previous
+        uses: styfle/cancel-workflow-action@0.9.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # TODO: Switch to WP-CLI stable (wp-cli.phar) once 2.5.0 is out.
+      - name: Install WP-CLI
+        run: |
+          curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli-nightly.phar
```

Comment:
Running nightly might be brittle. Is the intent to pick up the false positive bug fix sooner?

Reply:
As a maintainer of WP-CLI I can assure you the current nightly is quite stable :-)

The nightly contains some important bugfixes for newer PHP versions.

The next stable is already around the corner, so we can switch soon.

Reply:
> As a maintainer of WP-CLI I can assure you the current nightly is quite stable :-)

You may not always be on the project :P